src/site/content/en/blog/downloading-resources-in-html5-a-download/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Downloading resources in HTML5 - a[download]
+date: 2011-08-01
+updated: 2022-03-19 
+authors:
+  - ericbidelman
+description: |
+  Chrome now supports the HTML spec's new `download` attribute to `a` elements.
+tags:
+  - blog
+---
+
+Chrome now supports the HTML spec's new `download` attribute to `a` elements. When used, this attribute signifies that the resource it points to should be downloaded by the browser rather than navigating to it.
+
+From [Downloading Resources](https://html.spec.whatwg.org/dev/links.html#downloading-resources):
+
+{% Aside %}
+The `download` attribute, if present, indicates that the author intends the hyperlink to be used for downloading a resource. The attribute may have a value; the value, if any, specifies the default filename that the author recommends for use in labeling the resource in a local file system. There are no restrictions on allowed values, but authors are cautioned that most file systems have limitations with regard to what punctuation is supported in file names, and user agents are likely to adjust file names accordingly.
+
+The attribute can furthermore be given a value, to specify the filename that user agents are to use when storing the resource in a file system. This value can be overridden by the `Content-Disposition` HTTP header's `filename` parameter.
+{% endAside %}
+
+For example, clicking the following link downloads the .png as "MyGoogleLogo.png" instead of navigating to its `href` value: <a href="https://web-central.appspot.com/web/images/web-fundamentals-icon192x192.png" download="WebfundamentalsLogo">download me</a>. The markup is simple:
```

Comment:
```suggestion
For example, clicking the following link downloads the .png as "MyGoogleLogo.png" instead of navigating to its `href` value: <a href="https://web-central.appspot.com/web/images/web-fundamentals-icon192x192.png" download="WebfundamentalsLogo">download me</a>. The markup for this is as follows:
```

Reply:
This is a total stylistic nit, but we try to avoid phrases that trivialize tasks—this isn't something you need to keep an eye out for, but I tend to want to avoid this type of language if I find it.